extension-androidauto/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Add `Session.mapboxMapInstaller` and `Screen.mapboxMapInstaller` extension functions to create the installers. ([#1603](https://github.com/mapbox/mapbox-maps-android/pull/1603))
 
 ## Bug fixes 🐞
+* Fix map being stuck when app goes to background on Xiaomi devices. ([1638](https://github.com/mapbox/mapbox-maps-android/pull/1638))
```

Comment:
This changelog is actually part of the main maps sdk right?

There is nothing to release for the `extension-androidauto`, but there is an experimental `AnimationThreadController` api that can be used as part of the next maps release

Reply:
got it, updated `README.md` file instead.

Reply:
Why wasn't it added to the changelog for maps? Experimental apis are not added to maps changelogs?

Thinking there may be multiple changes related to this, and it will be tracked by Maps sdk versions

Reply:
This experimental api is needed for one use-case only (AA) and it's not related to SDK code - thus no changelog entry was added. We don't want this feature be used in any SDK use-cases and that's why there's no changelog entry.